### PR TITLE
Blacklist attach directly

### LINKED ISSUE #1825: Blacklisted posts in new peepr cannot be unhidden
see #1809 for why this is happening and #1811 for what can be done to fix it

### PR DESCRIPTION
**Pull request by hobinjk**
Fix #1825

I think I've forgotten how to program. I might clean this up a bit later but this works now

### CODE REVIEW

**Comment by hobinjk** [comment]
Notably I couldn't just check if .post contained a .post-form-form since that .post-form-form could in turn include a second .post

**Comment by hobinjk** [comment]
Actually this CSS query is rancid, I'm going to rewrite it the old way

File: Extensions/blacklist.js [hunk 1]
```diff
@@ -738,6 +762,13 @@ XKit.extensions.blacklist = new Object({
 		$(obj).addClass("xblacklist_blacklisted_post");
 		$(obj).prepend(block_excuse);
 
+		$(obj).find('.xblacklist_open_post').each(function() {
```
Comment (AprilSylph) [inline Extensions/blacklist.js:765]: could we do `$(obj).on('click', '.xblacklist_open_post', function() { $(obj).remove(); });`? now that unhiding the post is as simple as removing the excuse container

Comment (hobinjk) [reply Extensions/blacklist.js:765]: There's also the `$(m_div).removeClass("xblacklist_blacklisted_post");` at least. It doesn't quite work on the old dash but unhide_post is a good candidate for rewriting

Comment (AprilSylph) [reply Extensions/blacklist.js:765]: (assuming this works we could lose all the post id code above too, if im interpreting it right)

Comment (AprilSylph) [reply Extensions/blacklist.js:765]: do we still even need `xblacklist_blacklisted_post` for anything?

Comment (hobinjk) [reply Extensions/blacklist.js]: I've rewritten this code to be less paranoid/verbose

Comment (hobinjk) [reply Extensions/blacklist.js]: On the old dash it's used for styling

**Pull request merged**